DateTimePicker.js

```diff
@@ -11,6 +11,7 @@ import {
     View,
 } from 'react-native';
 import Button from 'react-native-osd-simple-button';
+import PropTypes from 'prop-types'
```

Comment:
Would you mind adding a `;` at the end of the line to keep up with linting rules please?